inst/extdata/correctionsataglance-example2.json

```diff
@@ -155,27 +99,424 @@
     "latitude": "38.9833375194952",
     "county": "LEAVENWORTH",
     "description": "Shows overview of correction information",
     "title": "Corrections At a Glance",
+    "longitute": "-94.9646893327574",
     "primaryParameter": "Turbidity, Form Neph._FNU@06892350",
     "primaryTsIdentifier": "2d8c0dcb028d45018441c32bd2be31a0",
-    "requestId": "6bcb57f3-fe09-4060-a576-6dd901ee6c79",
+    "requestId": "dc194c46-4bf6-46ca-b18f-192b95789ea8",
+    "contributingDrainageArea": "   59756",
+    "altitudeDatumCode": "NGVD29    ",
     "siteNumber": "06892350       ",
     "stationName": "KANSAS R AT DESOTO, KS",
     "coordinateDatumCode": "NAD27     ",
     "state": "KANSAS",
-    "stationId": "06892350",
-    "agency": "USGS ",
-    "drainageArea": "   59756",
-    "requestDateTime": "2015-12-24T11:00:08.732-06:00",
-    "nwisRaAuthToken": "3c60b972-1063-4867-94ff-3cd2671b96a5",
-    "longitute": "-94.9646893327574",
-    "contributingDrainageArea": "   59756",
-    "altitudeDatumCode": "NGVD29    ",
-    "startDate": "2010-01-01T00:00:00-06:00",
-    "ratingModelIdentifier": ""
-  }
+    "startDate": "2014-12-24T00:00:00-06:00",
+    "stationId": "06892350"
+  },
+  "fieldVisits": [
+    {
+      "locationIdentifier": "06892350",
+      "startTime": "2015-06-10T09:30:00.000-05:00",
+      "endTime": "2015-06-10T10:00:00.000-05:00",
+      "identifier": "25A8DF8A9FE4E747E0534CEB3D985776",
+      "isValid": true,
+      "lastModified": "2015-11-29T01:24:02.146-06:00",
+      "party": "mdm",
+      "remarks": "replaced EXO batteries, reset deployment.",
+      "weather": ""
+    },
+    {
+      "locationIdentifier": "06892350",
+      "startTime": "2014-12-31T10:00:00.000-06:00",
+      "endTime": "2014-12-31T10:30:00.000-06:00",
+      "identifier": "25A9607B56211253E0534CEB3D98967D",
+      "isValid": true,
+      "lastModified": "2015-11-29T01:23:24.116-06:00",
+      "party": "MDM",
+      "remarks": "Preemptive strike, proactive nitratax battery swap before batteries die over the long joyous holiday weekend break leading into 2015. Oh! what mysteries await us all in the new year; how will our equipment fare situated in the mighty Kansas River of DeSoto. Father Time tics on, and the existential conundrums shall continue to elude us until the moment of truth arrives, as so here is hoping that our little EXO and Nitratax cherish every moment and make the most out of it, taking the time to enjoy the little things like the majestic eagles\u0027 uncontrollable freedom of flight over the river, or the sound of swallow droppings plopping in the river all around the sonde installation, because we never know what debris may lay around the corner, or the shifting sand banks that may one day overcome the installation site and bring about change; but then, change is not always necessarily a bad thing. One last thing of note, the river was full of moving ice, still clear, ice free, moving water around the sonde and pier that it is situated behind.",
```

Comment:
We also may want to convert newlines "\n" to "<br/>" in the final HTML rendering. That may have fixed the rendering issues in the SRS report.